### ISSUE TITLE
Script for GRN analysis using SCENIC?

### ISSUE THREAD

**Issue by YushaLiu** [ding-lab/PanCan_snATAC_publication#5]
Thanks very much for providing this valuable dataset! I'm wondering where I can find scripts to reproduce the GRN analysis using SCENIC (regulon activity estimation and differential analysis)? Thanks for your help!

**Comment by nvterekhanova** [comment]
Hi @YushaLiu,

You can find scripts for running SCENIC in this folder: https://github.com/ding-lab/PanCan_snATAC_publication/tree/main/Fig3/Run_SCENIC

Nadezhda

**Comment by YushaLiu** [comment]
> Hi @YushaLiu,
> 
> You can find scripts for running SCENIC in this folder: https://github.com/ding-lab/PanCan_snATAC_publication/tree/main/Fig3/Run_SCENIC
> 
> Nadezhda

Thanks very much!

**Comment by YushaLiu** [comment]
Hi @nvterekhanova,
       I have a follow-up question about the cell type annotation in your snRNA-seq data. E.g., for a cell to be annotated as a T cell, do you require this cell to express all of the T cell marker genes (as listed in Supplementary table 1e), or express at least one of these marker genes? I also saw marker genes for Tregs in Table S1e (does that mean regulatory T cells?) but it seems none of the cells from your data were annotated as Tregs. 
      Could you provide more detailed description of how you arrived at the cell type annotations, particularly T cells (which we are interested in)? Or if you could point me to the scripts you used, that would also be super helpful. Thanks again for your help!

**Comment by nvterekhanova** [comment]
Hi @YushaLiu,

In this study we haven’t done a detailed cell type annotation for immune/stroma cells as the focus was primarily on cancer cells. So markers for T-regs (regulatory T-cells) as well as other markers for T-cells were used for annotating T-cells as a broad group here. 

For cell type annotation, we used clusters of cells, and labeled each cluster with cell type based on the expression values and the percentages of marker genes expressed. We examined this using the DotPlot function of the Seurat package. Also, in the supplements (Figs S2c-f) we have heatmaps showing averaged expression of markers across cell types, which shows relative marker specificity observed in each cell type.

Nadezhda
